test/algorithms/test_min_eigen_optimizer.py

```diff
@@ -245,6 +252,26 @@ def test_samples(self):
         self.assertAlmostEqual(result.fval, result.samples[0].fval)
         self.assertEqual(result.status, result.samples[0].status)
 
+    def test_bit_ordering(self):
+        """Test bit ordering"""
+        mdl = Model("docplex model")
+        x = mdl.binary_var('x')
+        y = mdl.binary_var('y')
+        mdl.minimize(x-2*y)
+        seed = 1234
+        qp = QuadraticProgram()
+        qp.from_docplex(mdl)
+        backend = QasmSimulator()
+        optimizer = SPSA(maxiter=100)
+        ry = TwoLocal(2, 'ry', 'cz', reps=3, entanglement='full')
+        quantum_instance = QuantumInstance(backend=backend,
+                                           seed_simulator=seed, seed_transpiler=seed)
+        vqe_mes = VQE(ry, optimizer=optimizer, quantum_instance=quantum_instance)
+        vqe = MinimumEigenOptimizer(vqe_mes)
+        result = vqe.solve(qp)
+        self.assertEqual(result.fval, -2)
+        self.assertListEqual(result.x.tolist(), [0, 1])
+
```

Comment:
Could you add some tests with samples? The optimal solution should have the highest probability.

Reply:
Sure. I added a test with samples as well.

Reply:
It would be better to follow `test_samples`, for example, comparison of x should be `np.testing.assert_array_almost_equal`.

Reply:
Why `result.raw_samples[0].x[0:2]` instead of `result.raw_samples[0].x`?

Reply:
Sure. I changed to use `np.testing.assert_array_almost_equal` and fixed the slice as you mentioned (I didn't need it in this test case)